README.md

```diff
@@ -36,7 +36,7 @@
 
 # 📖 [Book Collection] <a name="about-project"></a>
```

Comment:
- [ ] I think it would be great if you change the heading as well 🤩💪🏻since you wrote it here but you didn't change it above so kindly change it and add a description to your project as well 🤩🔥✨

![image](https://user-images.githubusercontent.com/58553711/228397378-14c62317-811a-4fd8-b2f8-eafb102c2803.png)